hyphengolang/noughts-and-crosses
# Mailing service issues

- add UI confirmation
- using agent lib
- created a lib directory
- rm commits
- fix for passwordless auth
- cleaning up event routing
- add pings for test purposes
- adding masked api routing
- api routes
- added a ping test on home page
- remove comments
- added test for ssr fetching and preflight
- no longer masking the API routes
- results type is being returned from message request
- removed versioning in api paths!
- added auth to check emails match on profile creation
- removed api versioning
- test not req
- cleanup
- gave a better error check
- moved all api helpers to agent
- tweaks to front end
- path aliasing
- merged results with events and rename
- refactoring services & messaging
- update renderfunc signature
- rename token client to just Client
- removed logging from handler
- remove final log
- temp log
- update to pages
- updates to apis in pkg dir
- include client_uri for service handler
- update to message handling
- broker now uses encConn
- remove compose + tweak api
- cleanup with component director
- include jwt_secret
- added fromPem option for jwt client
- update main file with new pem option
- key in test not production key, for testing purposes only
- ingore file update
- maining service
